app/Http/Middleware/CheckAdmin.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class CheckAdmin
+{
+    /**
+     * Handle an incoming request.
+     * 
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        /**
+         * Get the recent login user
+         */
+        $user = auth()->user();
+        if ($user && $user->is_admin) {
+
+            return $next($request);
```

Comment:
```suggestion
        if ($user && $user->is_admin) {
            return $next($request);
```
trong khối lệnh ko có dòng code nào khác trước return thì ko cần cách dòng trc return đâu :3